noaa/operations/default.json

```diff
@@ -0,0 +1,112 @@
+    {
+      "name": "index",
+      "operation-type": "index",
+      "bulk-size": 5000
+    },
+    {
+      "name": "range_query_range_field",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "range": {
+            "TRANGE": {
+              "gte": -10,
+              "lte": 10
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "range_query_range_field_in_conjunction_with_term_query",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "bool": {
+            "must": [
+              {
+                "term": {
+                  "station.keyword": "ASN00001018"
+                }
+              },
+              {
+                "range": {
+                  "TRANGE": {
+                    "gte": -10,
+                    "lte": 10
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    },
+    {
+      "name": "range_query_range_field_in_conjunction_with_terms_query",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "bool": {
+            "must": [
+              {
+                "terms": {
+                  "station.keyword": [
+                    "ASN00001018",
+                    "AQW00061705",
+                    "ARM00087453",
+                    "ARM00087576",
+                    "ASN00001006",
+                    "ASN00001007",
+                    "ASN00001010",
+                    "ASN00001013",
+                    "ASN00001018",
+                    "ASN00001019",
+                    "ARM00087121",
+                    "ASN00004032",
+                    "ASN00004046",
+                    "ASN00004019",
+                    "ASN00003105",
+                    "ASN00003100",
+                    "ASN00004083"
+                  ]
```

Comment:
> can we have a simple term query like the disjunction has?

I think you missed the `range_query_range_field_in_conjunction_with_term_query` query above this one?

> it might also help to have one conjunction with a range that matches most document
 and a term query that matches between 0.1 and 1%% of the index,

A weather station in this data set has at most 366 document which is 0,014% of the total amount of documents. So I think the 0.1 and 1% case is covered.

What query could be used for matching most of the docs, that on its own doesn't have a lot of overhead that could interfere with the benchmark? A term range? match_all ?